catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/contextual/ContextualEvaluationCriteriaImpl.java

```diff
@@ -53,7 +54,7 @@ public ContextualEvaluationCriteriaImpl(String criteria, boolean fuzzy,
 
         LOGGER.debug("criteria = {}", criteria);
         LOGGER.debug("textPaths:\n");
-        if (null != textPaths) {
+        if(null != textPaths) {
```

Comment:
Not part of the findbugs stuff per-se, but can we add an `if (LOGGER.isdebugEnabled())` check around the loop?